ci/run_cudf_examples.sh

```diff
@@ -7,38 +7,43 @@ EXITCODE=0
 trap "EXITCODE=1" ERR
 
 # Support customizing the examples' install location
-cd "${INSTALL_PREFIX:-${CONDA_PREFIX:-/usr}}/bin/examples/libcudf/" || exit
+cd "${INSTALL_PREFIX:-${CONDA_PREFIX:-/usr}}/bin/examples/libcudf" || exit
 
-cd basic || exit
+pushd basic || exit
 compute-sanitizer --tool memcheck basic_example
-cd ..
+popd || exit
 
-cd nested_types || exit
+pushd nested_types || exit
 compute-sanitizer --tool memcheck deduplication
-cd ..
+popd || exit
 
-cd strings || exit
+pushd strings || exit
 compute-sanitizer --tool memcheck custom_optimized names.csv
 compute-sanitizer --tool memcheck custom_prealloc names.csv
 compute-sanitizer --tool memcheck custom_with_malloc names.csv
-cd ..
+popd || exit
 
-cd string_transformers || exit
+pushd string_transformers || exit
 compute-sanitizer --tool memcheck compute_checksum_jit info.csv output.csv
 compute-sanitizer --tool memcheck extract_email_jit info.csv output.csv
 compute-sanitizer --tool memcheck extract_email_precompiled info.csv output.csv
 compute-sanitizer --tool memcheck format_phone_jit info.csv output.csv
 compute-sanitizer --tool memcheck format_phone_precompiled info.csv output.csv
 compute-sanitizer --tool memcheck localize_phone_jit info.csv output.csv
 compute-sanitizer --tool memcheck localize_phone_precompiled info.csv output.csv
-cd ..
+popd || exit
 
-cd parquet_io || exit
+pushd parquet_io || exit
 compute-sanitizer --tool memcheck parquet_io example.parquet
 compute-sanitizer --tool memcheck parquet_io example.parquet output.parquet DELTA_BINARY_PACKED ZSTD TRUE
 
 compute-sanitizer --tool memcheck parquet_io_multithreaded example.parquet
 compute-sanitizer --tool memcheck parquet_io_multithreaded example.parquet 4 DEVICE_BUFFER 2 2
-cd ..
+popd || exit
+
+pushd hybrid_scan || exit
+python datagen.py --output example.parquet --rows 1000 --row-group-size 200 --data-page-size 100
```

Comment:
This python script does take ~10s to run. Alternatively, I can just push the generated `example.parquet` file with the example of size ~650KiB. Thoughts @bdice @vyasr